ib-cli/installationiso/data/install-rhcos-and-restore-seed.sh

```diff
@@ -6,27 +6,42 @@ seed_image=${1:-$SEED_IMAGE}
 seed_version=${2:-$SEED_VERSION}
 installation_disk=${3:-$INSTALLATION_DISK}
 lca_image=${4:-$LCA_IMAGE}
+extra_partition_start=${5:-$EXTRA_PARTITION_START}
```

Comment:
Why there's a 5 here? Should be "use directory" no?

Reply:
It's the 5th argument from the command line in case you call the install-rhcos-and-restore-seed.sh script :sweat_smile: 

In the systemd unit we feed the data through env variables, but for testing it is useful :)